.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
```

Comment:
Sorry if I wasn't clear. I still want the default to be 4. We just need to have 2 for `*.js` files only.

Reply:
no problem. PR updated.